src/main/java/org/springframework/integration/aws/outbound/KplMessageHandler.java

```diff
@@ -99,6 +100,10 @@ public class KplMessageHandler extends AbstractAwsMessageHandler<Void> implement
 
 	private volatile ScheduledFuture<?> flushFuture;
 
+	private long maxRecordsInFlight = 0;
+
+	private int maxRecordInFlightsSleepDurationInMillis = 100;
```

Comment:
I don't think there is a reason to mention in the property name that it is "in millis".
Just `maxInFlightRecordsDuration`.
The rest we would just mention in the respective JavaDocs.

Reply:
Agree. Fixed.